core/src/main/java/nl/nn/adapterframework/senders/SenderSeries.java

```diff
@@ -120,13 +120,6 @@ public boolean consumesSessionVariable(String sessionKey) {
 		return false;
 	}
 
-
-
-	@Deprecated // replaced by registerSender, to allow for multiple senders in XSD. Method must be present, as it is used by Digester
-	public final void setSender(ISender sender) {
-		registerSender(sender);
-	}
-
```

Comment:
Ik denk dat dit mogelijk fouten oplevert omdat de digester zelf geen onderscheid maakt tussen set en register en (volgens mij) alleen de eerste methode pakt. De Frank!Doc gebruikt ze wel beide. Hier is ooit een issue voor aangemaakt #1601. Mocht je ideeën hebben, heel graag!

Reply:
Ah thanks! Dat was dus het probleem denk ik, waar ik vrijdagmiddag mee aan het puzzelen was. Ga ik maandag verder uitzoeken.

Reply:
Kan je de digester dan niet vertellen om de `registerSender` method te gebruiken?